### DISCUSSION TITLE
Integration vs Merging of Datasets

### DISCUSSION THREAD

**Discussion by andrewwbutler** [satijalab/seurat#3998] [Q&A] [answered]
When should I use Seurat integration instead of merging?

**Comment by satijalab** [comment, marked as answer]
The Seurat integration procedure aims to identify shared cell populations across different datasets, and ensure that they group together after integration. This enables the frequency and gene expression profiles of these populations to be effectively compared in downstream analysis. 

However, it may not be necessary to perform integrative analysis, particularly in cases where there are not expected to be substantial batch effects or biological differences across samples. In these cases, users can begin each analysis by merging the datasets together (using the `merge` command), and proceed with the standard analysis workflow. Users can apply clustering and visualization to see if cells from the same biological subpopulations group together across datasets. If so, there is no need to perform integrative analysis. However, if cells cluster primarily by their dataset of origin as opposed to their biological state, we suggest applying our Seurat integration workflow

**Reply by schmerlain14** [reply]
Using this advice at which point would you suggest to normalize (use SCTransform)? Before integration/merge or after?
